docs/source/onnx_dynamo_memory_usage.rst

```diff
@@ -0,0 +1,111 @@
+Understanding TorchDynamo-based ONNX Exporter Memory Usage
+==========================================================
+The previous TorchScript-based ONNX exporter would execute the model once to trace its execution, which could cause it to run out of
+memory on your GPU if the model's memory requirements exceeded the available GPU memory. This issue has been addressed with the new
+TorchDynamo-based ONNX exporter."
+
+The TorchDynamo-based ONNX exporter leverages `FakeTensorMode <https://pytorch.org/docs/stable/torch.compiler_fake_tensor.html>`_ to
+avoid performing actual tensor computations during the export process. This approach results in significantly lower memory usage
+compared to the TorchScript-based ONNX exporter.
+
+Below is an example demonstrating the memory usage difference between TorchScript-based and TorchDynamo-based ONNX exporters.
+In this example, we use the HighResNet model from MONAI. Before proceeding, please install it from PyPI:
+
+.. code-block:: bash
+
+   pip install monai
+
+
+PyTorch offers a tool for capturing and visualizing memory usage traces. We will use this tool to record the memory usage of the two
+exporters during the export process and compare the results. You can find more details about this tool on
+`this page <https://pytorch.org/docs/stable/torch_cuda_memory.html>`_.
```

Comment:
I usually avoid ``this page`` as anchors because search engines cannot use tha information to help users.